server/src/main/java/io/crate/action/sql/Session.java

```diff
@@ -167,6 +176,9 @@ public Session(int sessionId,
                    CoordinatorSessionSettings sessionSettings,
                    Runnable onClose) {
         this.id = sessionId;
+        this.handlerNode = handlerNode;
+        this.connectionProperties = connectionProperties;
+        this.timeStarted = System.currentTimeMillis();
```

Comment:
Maybe `timeCreated` ? Or initiated/connected.

`started` seems more likely to confuse with the start of a statement/operation. A session itself doesn't start in a way.

Reply:
I agree, it was stated like that in the issue, so `timeCreated` sounds better, I'll also change the column name of the table. thx!